sql/moz-fx-data-glam-prod-fca7/glam_etl/glam_fog_release_aggregates_v1/metadata.yaml

```
@@ -7,9 +7,8 @@ labels:
   incremental: false
   owner1: efilho@mozilla.com
 scheduling:
-  dag_name: bqetl_glam_refresh_aggregates
+  dag_name: bqetl_glam_refresh_aggregates_fog_release
   date_partition_parameter: null
   depends_on:
-  - task_id: export_firefox_desktop_glam_release
-    dag_name: glam_fog
-    execution_delta: 6h
+  - task_id: fog_release_done
+    dag_name: glam_fog_release
```

This should have `execution_delta` since `glam_fog_release` runs at `0 10 * * 6` 
```suggestion
    dag_name: glam_fog_release
    execution_delta: 8h

```